marklogic-data-hub-central/build.gradle

```diff
@@ -186,6 +188,63 @@ test {
 	useJUnitPlatform()
 }
 
+ospackage {
+    packageName = 'hub-central'
+    version = "${project.version}"
+    release = 1
```

Comment:
What does "release" mean/affect here? When would we change this to some other value?

Reply:
It refers to the RPM release version. 
We increment the release version only if the payload (hub-central) of the RPM remains the same but RPM itself undergoes some change.

Meaning if we publish the RPM with hub-central 5.3.0 and if the RPM scripts have some issue and need to be updated then we will increment the release number for the RPM.

The RPM name is generally of the type NVR (Name-Version-Release)
So in the above case it becomes 
hub-central-5.3.0-1.noarch.rpm -> hub-central-5.3.0-2.noarch.rpm

Reply:
Got it, thanks. That may be worth adding as a comment above the "release" line, as I imagine other developers down the road will wonder how "version" and "release" interact.